src/DynamoApplications/StartupUtils.cs

```diff
@@ -167,21 +167,36 @@ public static void PreloadShapeManager(ref string geometryFactoryPath, ref strin
             geometryFactoryPath = preloader.GeometryFactoryPath;
             preloaderLocation = preloader.PreloaderLocation;
         }
-
-        /// <summary>
-        /// Use this overload to construct a DynamoModel in CLI context when the location of ASM to use is known, host analytics info is known and you want to set data paths.
-        /// </summary>
-        /// <param name="asmPath">Path to directory containing geometry library binaries</param>
-        /// <param name="userDataFolder">Path to be used by PathResolver for UserDataFolder</param>
-        /// <param name="commonDataFolder">Path to be used by PathResolver for CommonDataFolder</param>
-        /// <param name="info">Host analytics info specifying Dynamo launching host related information.</param>
-        /// <returns></returns>
-        public static DynamoModel MakeCLIModel(string asmPath, string userDataFolder, string commonDataFolder, HostAnalyticsInfo info = new HostAnalyticsInfo())
+        public static DynamoModel PrepareModel(
```

Comment:
you cannot break these public api signatures.

Reply:
@chubakueno I guess a better option should be creating a overloaded for `public static DynamoModel MakeModel` which receives  `string CLIlocale ` as parameter so in this way you won't need to modify the existing public method (I think default argument is also **NOT** an option).

Reply:
Done, though there are now two(previously there was one) unused overloads of the methods.

Reply:
@chubakueno I can see that now we have three MakeModel() methods and only two of them are calling PrepareModel(), is possible to also use PrepareModel() in the missing one?
![image](https://github.com/user-attachments/assets/a4e30dfb-71d4-45e3-b1ec-f80b707f396c)

Reply:
Done. Behavior before was not identical to the other overloads so PrepareModel wasn't an equivalent refactor (no calls to `AssignHostPathAndIPathResolver`, `SetUICulture`, `OnDetectLanguage`, `OnRequestUpdateLoadBarStatus`), but that MakeModel is only used by integration tests and the lack of those calls is more likely to be a bug so i think it's a good idea that all exhibit the same behavior.